wgsl/index.bs

```diff
@@ -865,135 +1031,47 @@ It is defined recursively in the following table:
           <!-- Vulkan base alignment: 4 times scalar aligment -->
   <tr algorithm="alignment of an array">
       <td>array<|T|,|N|>
-      <td>|Align|(|T|,`storage`)
-      <td>[=roundUp=](16, |Align|(|T|,`uniform`))
+      <td>[=RequiredAlignOf=](|T|, |C|)
```

Comment:
I find these descriptions a bit confusing. The heading makes clear which storage class is used so it might be better to change this to: `[=AlignOf=](|T|)` for storage and `roundUp([=AlignOf=](|T|), 16)` for uniform. This also makes it clearer that default rules satisfy `storage` storage requirements.

Reply:
Similar change for runtime arrays and structs too.

Reply:
I found trying to apply this change extremely fiddly for the bit below:

```
All structure members of type |T| must have a byte offset from the start of the
structure that is a multiple of the [=RequiredAlignOf=](|T|, |C|) for the storage
class |C|:
```

Going to have to revisit this when my brain is in a better state

Reply:
Simplified this table to refer back to `AlignOf()` as suggested.